demo/admin/src/App.tsx

```diff
@@ -102,151 +103,164 @@ class App extends React.Component {
-                                                >
-                                                    <React.Fragment>
-                                                        <GlobalStyle />
-                                                        <ContentScopeProvider>
-                                                            {({ match }) => (
-                                                                <Switch>
-                                                                    {/* @TODO: add preview to contentScope once site is capable of contentScope */}
-                                                                    <Route
-                                                                        path={`${match.path}/preview`}
-                                                                        render={(props) => <SitePreview {...props} />}
-                                                                    />
-                                                                    <Route
-                                                                        render={(props) => (
-                                                                            <MasterLayout headerComponent={MasterHeader} menuComponent={MasterMenu}>
-                                                                                <Switch>
-                                                                                    <RouteWithErrorBoundary
-                                                                                        path={`${match.path}/dashboard`}
-                                                                                        component={Dashboard}
-                                                                                    />
-                                                                                    <RouteWithErrorBoundary
-                                                                                        path={`${match.path}/project-snips/main-menu`}
-                                                                                        component={MainMenu}
-                                                                                    />
-                                                                                    <RouteWithErrorBoundary
-                                                                                        path={`${match.path}/pages/pagetree/:category`}
-                                                                                        render={({ match: { params } }) => {
-                                                                                            const category = urlParamToCategory(params.category);
+                        <DependenciesConfigProvider
+                            entityDependencyMap={{
+                                Page: Page,
+                                Link: Link,
```

Comment:
Can you also provide an example in demo how this would look for Product or News? And do you also need DamFile in here?

Reply:
> Can you also provide an example in demo how this would look for Product or News?

I'll do that

> And do you also need DamFile in here?

No, it's automatically added in the library: https://github.com/vivid-planet/comet/blob/9a1d7d5f4f28359f0eeabc959b50f04959761447/packages/admin/cms-admin/src/dependencies/DependenciesConfig.tsx#L18-L21

Reply:
> Can you also provide an example in demo how this would look for Product or News?

I did this in https://github.com/vivid-planet/comet/pull/1023/commits/2c2b74ce0b2ab545763d9860e98114069b5464d7 for News. It's pretty straight-forward. 

I didn't do it for Product because saving the products doesn't work currently and I don't want to debug that now.

I also added basic docs with the steps necessary in https://github.com/vivid-planet/comet/pull/1023/commits/35434603c5cc8b87d3b4e6910c4da03138365ebd